os-server/src/main/java/app/openschool/course/discussion/util/ValidationHandler.java

```diff
@@ -23,4 +30,10 @@ public void validateSearchQuery(String searchQuery) {
           messageSource.getMessage("discussion.mentor.question.query.size", null, Locale.ROOT));
     }
   }
+
+  public boolean checkUserEnrollment(Long enrolledCourseId, String userEmail) {
+    Optional<EnrolledCourse> enrolledCourse = enrolledCourseRepository.findById(enrolledCourseId);
```

Comment:
please create a new method in the repository that will return a result if the user enrolled or not

Reply:
Understood, will fix it.